frontend/src/components/ChatMessage.tsx

```
@@ -1,20 +1,84 @@
-    sender: "user" | "ai";
-    text: string;
-  };
+  message: Message;
 }
 
+const ToolResultMessage: React.FC<{ toolName: string; result: any }> = ({ toolName, result }) => {
+  switch (toolName) {
+    case "list_files":
+    case "list_snapshots":
+      if (result.error) return <div className="error-message">Error: {result.error}</div>;
+      const items = result.files || result.snapshots || [];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)